VAMobile/src/components/PhotoUpload.tsx

```diff
@@ -0,0 +1,171 @@
+import { Image, Pressable, PressableProps } from 'react-native'
+import { ImagePickerResponse, launchCamera, launchImageLibrary } from 'react-native-image-picker'
+import { useTranslation } from 'react-i18next'
+import React, { FC, ReactNode, useEffect, useState } from 'react'
+import styled from 'styled-components'
+
+import { NAMESPACE } from 'constants/namespaces'
+import { VAIcon } from './index'
+import { themeFn } from 'utils/theme'
+import { useDestructiveActionSheet, useShowActionSheet } from 'utils/hooks'
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import Box, { BoxProps } from './Box'
+import TextView from './TextView'
+import theme from 'styles/themes/standardTheme'
+
+type PhotoUploadProps = {
+  /** width of the photo */
+  width: number
+  /** height of the photo */
+  height: number
+  /** imagePickerResponse with asset to style for component and fileSize */
```

Comment:
Don't understand this comment. Should it go somewhere else?

Reply:
Yup, copy paste fail. Removed.